.gitignore

```diff
@@ -1,5 +1,62 @@
-SmartDeviceLink.framework
-MobileWeather.xcodeproj/project.xcworkspace/xcshareddata
-MobileWeather.xcodeproj/project.xcworkspace/xcuserdata
-MobileWeather.xcodeproj/xcshareddata
-MobileWeather.xcodeproj/xcuserdata
+# Xcode
+#
+# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
+
+## Build generated
+build/
+DerivedData/
+
+## Various settings
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata/
+
+## Other
+*.moved-aside
+*.xcuserstate
+
+## Obj-C/Swift specific
+*.hmap
+*.ipa
+*.dSYM.zip
+*.dSYM
+
+# CocoaPods
+#
+# We recommend against adding the Pods directory to your .gitignore. However
+# you should judge for yourself, the pros and cons are mentioned at:
+# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
+#
+# Pods/
+
```

Comment:
Are we just using **CocoaPods** or are we including the **Pods** directory in the project also?